crates/napi/src/bindgen_runtime/mod.rs

```diff
@@ -46,11 +45,6 @@ pub unsafe extern "C" fn raw_finalize_unchecked<T>(
         "Rc strong count is: {}, it should be 1 or 2",
         rc_strong_count
       );
-      assert!(
-        rc_weak_count == 0,
-        "Rc weak count is: {}, it should be 0",
-        rc_weak_count
-      );
```

Comment:
I wasn't sure exactly what this assertion was for (there didn't ever seem to be any weak references before?), but it seems unnecessary as long as there are no strong references.

Reply:
Yes, it doesn't seem necessary for now